### ISSUE TITLE
Армблейд сломался...баг?

### ISSUE THREAD

**Issue by artur0116** [TauCetiStation/TauCetiClassic#10029]
#### Подробное описание проблемы
Играя за генокрада, прожав армблейд он не появился.Но химикаты потратились.Не писал бы этот ишью если бы не жалоба от игрока в чате, у которого была похожая проблема "после фикса молота".Я не увидел армблейд не в спрайте у своей руки, не тыкая по человечку.
#### Что должно было произойти
появление армблейда после юзание спелла
#### Что произошло на самом деле
ничево...армблейд не появился! но химикаты потратились.
#### Как повторить
прожать армблейд после использования крика,(Как было у меня)
#### Дополнительная информация:
нету.

**Comment by BlankCringe** [comment]
ай ди раунда в доп информацию кинь

**Comment by artur0116** [comment]
> ай ди раунда в доп информацию кинь чтобы кодер мог увидеть в чём проблем

не знаю, посмотрите последний раунд с генокрадами на первом, там вот тот момент.Кукла Jad Shock, примерно где то в 7 часов раунд был

**Comment by LoTaRk-FroG** [comment]
Играл в том раунде за генокрада, всё работало, скорее всего ты случайно 2 раза кликнул по кнопке.

**Comment by artur0116** [comment]
> Играл в том раунде за генокрада, всё работало, скорее всего ты случайно 2 раза кликнул по кнопке.

Нет, я именно использовал крик и прожал армблейд. У меня было 50 химикатов, 50-25-20, у меня оставалось 5 химикатов. Но я бы не стал писать этот ишью если бы данная проблема у меня была только. Парень в ГОСТ чате отписал о схожем баге

**Comment by LoTaRk-FroG** [comment]
> > Играл в том раунде за генокрада, всё работало, скорее всего ты случайно 2 раза кликнул по кнопке.
> 
> Нет, я именно использовал крик и прожал армблейд. У меня было 50 химикатов, 50-25-20, у меня оставалось 5 химикатов. Но я бы не стал писать этот ишью если бы данная проблема у меня была только. Парень в ГОСТ чате отписал о схожем баге

Нажатие кнопки спауна армблейда не тратит химикаты если армблейд уже в руке.
